app/src/main/res/values-kab/strings.xml

```diff
@@ -775,4 +814,7 @@
     <!-- Text for the menu button to remove a top site -->
     <string name="remove_top_site">Kkes</string>
 
-    </resources>
+    <!-- Text for context menu additional note.
+    %1$s is a placeholder for the app name. -->
+    <string name="contextmenu_erased_images_note2">Tugniwin yettwaskelsen akked yettwabḍan &lt;b&gt;ur ttwakkasent ara&lt;/b&gt; mi ara ad tekkseḍ %1$ azray.</string>
```

Comment:
@Delphine This string needs updating. I suggested the change here: https://pontoon.mozilla.org/kab/focus-for-android/mozilla-mobile/focus-android/app/src/main/res/values/strings.xml/?search=contextmenu_erased_images_note2&string=231039

Reply:
Approved the change in Pontoon! thanks for catching that

Reply:
> @Delphine This string needs updating. I suggested the change here: https://pontoon.mozilla.org/kab/focus-for-android/mozilla-mobile/focus-android/app/src/main/res/values/strings.xml/?search=contextmenu_erased_images_note2&string=231039

TIL we can fix these errors in Pontoon ourselves. 🤯

Reply:
@jonalmeida: Haha, yes - I still need to approve those suggested fixes in Pontoon though, once someone submits it! But I'm thinking that for "just" placeholder l10n errors, I could give a couple people from mobile team the rights in Pontoon to approve the fixes so that they can approve them themselves 🤔 (perhaps you and @mcarare, for example?)
Maybe these issues (ie, placeholder errors) would be the "Pontoon super power" you use that for? Would that possibly help speed up the process? (and then you can tag me when there are other issues needing to be fixed in Pontoon, that you're not sure about)
Thoughts?
And note that I'm fine with approving myself - just trying to find an simpler process if possible :)

Reply:
One case where I'd necessarily need to be CCed in case of placeholder errors, for example, would be in the case where the localizer completely omits the placeholder - so I would have to contact them to know exactly where it has to go in the string

Reply:
> Maybe these issues (ie, placeholder errors) would be the "Pontoon super power" you use that for? Would that possibly help speed up the process, instead of having to wait for me to unblock you? (and then you can tag me when there are other issues needing to be fixed in Pontoon, that you're not sure about)
> Thoughts?
> And note that I'm fine with approving myself - just trying to find an simpler process if possible :)

This providing suggestions on Pontoon sounds like a good enough solution. Having it verified through you is safer.